internal/p2p/peermanager_test.go

```diff
@@ -1311,7 +1311,7 @@ func TestPeerManager_Ready(t *testing.T) {
 	require.Equal(t, p2p.PeerStatusDown, peerManager.Status(a.NodeID))
 
 	// Marking a as ready should transition it to PeerStatusUp and send an update.
-	peerManager.Ready(ctx, a.NodeID)
+	peerManager.Ready(ctx, a.NodeID, nil)
```

Comment:
Is there a practical way to verify that nontrivial channel maps are also correctly propagated?

Reply:
added a unittest